<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{
border-image: url(:/Image/webp.png);
}
QToolButton{
font: 18pt &quot;文泉驿等宽微米黑&quot;;
border:2px none;
border-radius:8px;
color: rgb(0, 0, 0);
border-color: rgb(53, 156, 212);

}
QToolButton:pressed{
font: 16pt &quot;文泉驿等宽微米黑&quot;;
background-color:rgb(85, 170, 255);
}
</string>
   </property>
   <widget class="QToolButton" name="tBtn_hostCheck">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>100</width>
      <height>90</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>主机自检</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/Image/ooopic_1524532085.png</normaloff>:/Image/ooopic_1524532085.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_hostSet">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>100</width>
      <height>90</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>主机设置</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/Image/ooopic_1524532052.png</normaloff>:/Image/ooopic_1524532052.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_record">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>100</width>
      <height>90</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>历史记录</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/Image/record.png</normaloff>:/Image/record.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_hostInfo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>100</width>
      <height>90</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>主机信息</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/Image/ooopic_1524532068.png</normaloff>:/Image/ooopic_1524532068.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_devInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>100</width>
      <height>90</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>监控信息</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/Image/ooopic_1524532108.png</normaloff>:/Image/ooopic_1524532108.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_user">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>100</width>
      <height>90</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>用户登录</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/Image/ooopic_1524532936.png</normaloff>:/Image/ooopic_1524532936.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>80</y>
      <width>546</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(255, 255, 255);
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>消防设备电源监控系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>425</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>445</y>
      <width>340</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtn_quit">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>425</y>
      <width>136</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>程序退出</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/Image/running.png</normaloff>:/Image/running.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
